code/sonalyze/sonarlog/sysinfo.go

```diff
@@ -0,0 +1,26 @@
+package sonarlog
+
+import (
+	"encoding/json"
+	"io"
+
+	"go-utils/config"
+)
+
+// Sysinfo records appear in sequence in the input without preamble/postamble or separators.
+
```

Comment:
Add a comment here that if an error is encountered we will return the records successfully parsed along with an error, but that there is no ability to skip erroneous records and continue going after an error has been encountered.